### PR TITLE
Docs: Add No basic role feature description

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -89,3 +89,18 @@ We are gradually rolling out our new browse dashboards user interface. With this
...
+## Authentication and authorization
+
+### No Basic Role
+
+<!-- Eric Leijonmarck -->
+<!-- OSS, Enterprise -->
+
+Generally available in all editions of Grafana.
+
+We're excited to introduce the "No Basic Role," a new basic role with no permissions. A basic role in Grafana dictates the set of actions a user or entity can perform, known as permissions. This new role is especially beneficial if you're aiming for tailored, customized RBAC permissions for your service accounts or users. You can set this as a basic role through the API or UI.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:102]:
```suggestion
We're excited to introduce the "No basic role," a new basic role with no permissions. A basic role in Grafana dictates the set of actions a user or entity can perform, known as permissions. This new role is especially beneficial if you're aiming for tailored, customized RBAC permissions for your service accounts or users. You can set this as a basic role through the API or UI.
```

**Review by imatwawana — COMMENTED** [review]
Made some small edits including capitalization fixes and a new linking style. Thanks for moving this over!

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -89,3 +89,18 @@ We are gradually rolling out our new browse dashboards user interface. With this
...
+
+Previously, permissions were granted based on predefined sets of capabilities. Now, with the "No Basic Role," you have the flexibility to be even more granular.
+
+For more details on basic roles and permissions, refer to the [documentation]{{< relref "../../administration/roles-and-permissions#organization-roles" >}}.
```
[the remark this answers is not in the document]
Comment (eleijonmarck) [reply docs/sources/whatsnew/whats-new-next/index.md:106]: so should i specify the <GRAFANA VERSION> or this something new? 🤔

[the remark this answers is not in the document]
Comment (eleijonmarck) [reply docs/sources/whatsnew/whats-new-next/index.md]: i like the new linking

**Comment by eleijonmarck** [comment]
@imatwawana anything else I would need to do?

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -89,3 +89,18 @@ We are gradually rolling out our new browse dashboards user interface. With this
 To learn more, refer to the following video demo.
 
 {{< video-embed src="/media/docs/grafana/2023-09-11-New-Browse-Dashboards-Enablement-Video.mp4" >}}
+
+## Authentication and authorization
+
+### No basic role
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:95]: In the on-prem What's new, we have sections, but for now in the Cloud version, we don't. 
```suggestion
## No basic role
```

**Review by imatwawana — APPROVED** [review]
Thanks so much for adding this new PR! I made one change to the headings to conform to the formatting for the Cloud what's new, but otherwise, this looks good!

**Pull request merged**